packages/conjure-client/src/fetchBridge/blobReadableStreamAdapter.ts

```diff
@@ -30,7 +30,14 @@ export function blobToReadableStream(blobPromise: Promise<Blob>): ReadableStream
             blobPromise.then(blob => reader.readAsArrayBuffer(blob));
         },
     };
-    return typeof ReadableStream === "undefined"
-        ? (new PonyfilledReadableStream<Uint8Array>(underlyingSource) as ReadableStream<Uint8Array>)
-        : new ReadableStream(underlyingSource);
+    if (process.env.MODERN) {
+        // We must cast here so we can stay on ancient typescript version 3.x!
```

Comment:
Can we remove the cast since this PR upgrades typescript?

Reply:
Thanks for the catch. Wrote that and then later decided to update typescript.